features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/tracking/AndroidXFragmentLifecycleCallbacks.kt

```diff
@@ -77,11 +87,19 @@ internal open class AndroidXFragmentLifecycleCallbacks(
         }
     }
 
-    override fun onFragmentPaused(fm: FragmentManager, f: Fragment) {
-        super.onFragmentPaused(fm, f)
-        componentPredicate.runIfValid(f, internalLogger) {
-            val key = resolveKey(it)
-            rumMonitor.stopView(key)
+    @MainThread
+    override fun onFragmentStopped(fm: FragmentManager, f: Fragment) {
```

Comment:
if we do this here, shouldn't we move the `startView` on the `onStart` also ? This will lower the gap.

Reply:
No, because the Fragment can be started but not visible yet (e.g.: in a view pager, the fragment is not resumed until visible but it can be started long before that)